### PR TITLE
PLAT-367: get rid of IgnoreAllErrors from most of the tests

### PR DESCRIPTION
**Pull request by sexyhamster**
The only place that still holds IgnoreError are network integration tests, which would be reworked

### CODE REVIEW

File: ledger-core/application/api/availability_test.go [hunk 1]
```diff
@@ -39,7 +40,12 @@ func waitForStatus(t *testing.T, nc *api.NetworkChecker, expected bool) {
 }
 
 func TestAvailabilityChecker_UpdateStatus(t *testing.T) {
-	instestlogger.SetTestOutputWithIgnoreAllErrors(t)
+	instestlogger.SetTestOutputWithErrorFilter(t, func(s string) bool {
+		expectedError := strings.Contains(s, "no response or bad StatusCode") ||
+			strings.Contains(s, "Can't decode body: invalid character '}'") ||
+			strings.Contains(s, "connection refused")
```
Comment (aalekseev-ru) [inline ledger-core/application/api/availability_test.go:46]: maybe move those strings to constants?

Comment (sexyhamster) [reply ledger-core/application/api/availability_test.go:46]: what do you mean?

Comment (sexyhamster) [reply ledger-core/application/api/availability_test.go:46]: they are not constants. That couple of expected errors that are in this test

**Comment by ita-sammann** [comment]
test

File: ledger-core/server/internal/headless/components_test.go [hunk 2]
```diff
@@ -20,7 +21,12 @@ import (
 )
 
 func TestComponents(t *testing.T) {
-	instestlogger.SetTestOutputWithIgnoreAllErrors(t)
+	instestlogger.SetTestOutputWithErrorFilter(t, func(s string) bool {
+		if strings.Contains(s, "Failed to export to Prometheus: cannot register the collector: duplicate metrics collector registration attempted") {
```
Comment (cyraxred) [inline ledger-core/server/internal/headless/components_test.go:25]: also happens
"Failed to export to Prometheus: cannot register the collector: descriptor Desc ..."
so I suggest to use 
"Failed to export to Prometheus: cannot register the collector"

**Pull request merged**